bentoml/server/api_server.py

```diff
@@ -536,52 +556,120 @@ def handle_ml_predict(self):
             'Authorization': 'Bearer ' + access_token,
             'Content-Type': 'application/ld+json'
         }
-        MLMODEL_UUID = self.ngsild_ml_model_id
+        
+        WEATHER_UUID = 'urn:ngsi-ld:WeatherObserved:Pegomas'
```

Comment:
don't get why you are hardcoding this, should not history simply be retrieved from the notified entity?

Reply:
Yes. But in the context of doing a demo, I didn't want to have to update with dummy data a 'live' entity.